storage/src/main/java/tech/pegasys/artemis/storage/MapDbDatabase.java

```diff
@@ -67,15 +73,58 @@
   private final ConcurrentNavigableMap<UnsignedLong, Set<Bytes32>> hotRootsBySlotCache =
       new ConcurrentSkipListMap<>();
 
+  private static void validateDatabaseVersion(File databaseVersionFile) {
+    try {
+      if (databaseVersionFile.exists()) {
+        Scanner contents = new Scanner(databaseVersionFile, UTF_8.name());
+        String ver = contents.nextLine();
```

Comment:
Would `Files.readString(databaseVersionFile.toPath()).trim()` be simpler?  Would also be slightly stricter by requiring no other content in the file rather than accepting anything that happens to have 1 on the first line.

Reply:
sounds good.